src/spread/mult-obj-null-empty-undefined.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Object Spread operator following other arguments with null, undefined and empty object
+template: default
+esid: pending
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- args
+{a: 1, ...null, b: 2, ...undefined, c: 3, ...{}, d: 4}
+//- params
+obj
+//- body
+assert.sameValue(obj.a, 1);
+assert.sameValue(obj.b, 2);
+assert.sameValue(obj.c, 3);
+assert.sameValue(obj.d, 4);
+assert.sameValue(Object.keys(obj).length, 4);
```

Comment:
if you include the harness/compareArray.js, you can also assert the order they are added. To avoid false positives, resort the order you include a, b, c, and d.